htdocs/admin/barcode.php

```diff
@@ -441,6 +441,47 @@
  * Other options
  */
 print load_fiche_titre($langs->trans("OtherOptions"), '', '');
+// Init barcode for thirdparty (manual trigger)
+if (isModEnabled('societe')) {
+	print '<div style="color:red;">DEBUG: Barcode Thirdparty Init Section Loaded</div>';
+	print load_fiche_titre($langs->trans("InitThirdpartyBarcodes"), '', '');
+
+	if (GETPOST('action', 'aZ09') == 'init_barcode_empty_thirdparty') {
```

Comment:
The code for action must be in the controler section so after the comment /* Action and before the comment /* View.
Thanks to move it.